x-pack/plugins/ml/public/embeddables/anomaly_explorer/explorer_input_resolver.ts

```diff
@@ -0,0 +1,259 @@
+          });
+
+          let influencersFilterQuery: any;
+          try {
+            influencersFilterQuery = processFilters(filters, query);
+          } catch (e) {
+            // handle query syntax errors
+            setError(e);
+            return of(undefined);
+          }
+
+          const bounds = anomalyTimelineService.getTimeBounds();
+
+          // Can be from input time range or from the timefilter bar
+          const selections: AppStateSelectedCells = {
+            lanes: [OVERALL_LABEL],
+            times: [bounds.min?.unix()!, bounds.max?.unix()!],
+            type: SWIMLANE_TYPE.OVERALL,
+          };
+
+          const selectionInfluencers = getSelectionInfluencers(selections, viewBySwimlaneFieldName);
+
+          const jobIds = getSelectionJobIds(selections, explorerJobs);
+
+          const bucketInterval = timeBuckets.getInterval();
+
+          const timeRange = getSelectionTimeRange(selections, bucketInterval.asSeconds(), bounds);
+          const explorer$ = forkJoin({
+            combinedJobs: anomalyExplorerService.getCombinedJobs(jobIds),
+            anomalyChartRecords: loadDataForCharts(
+              jobIds,
+              timeRange.earliestMs,
+              timeRange.latestMs,
+              selectionInfluencers,
+              selections,
+              influencersFilterQuery,
+              false
+            ),
+          }).pipe(
+            switchMap(({ combinedJobs, anomalyChartRecords }) => {
+              const combinedJobRecords: Record<
+                string,
+                CombinedJob
+              > = (combinedJobs as CombinedJob[]).reduce((acc, job) => {
+                return { ...acc, [job.job_id]: job };
+              }, {});
+
+              return forkJoin({
+                chartsData: from(
+                  anomalyExplorerService.getAnomalyData(
+                    undefined,
+                    combinedJobRecords,
+                    embeddableContainerWidth,
+                    anomalyChartRecords,
+                    timeRange.earliestMs,
+                    timeRange.latestMs,
+                    timefilter,
+                    severityValue,
+                    maxSeriesToPlot
+                  )
+                ),
+              });
+            })
+          );
+          return explorer$;
+        }),
+        catchError((e) => {
+          setError(e.body);
+          return of(undefined);
+        })
+      )
+      .subscribe((results) => {
+        if (results !== undefined) {
+          setError(null);
+          setChartsData(results.chartsData);
+          setIsLoading(false);
+        }
+      });
+
+    return () => {
+      subscription.unsubscribe();
+    };
+  }, []);
+
+  useEffect(() => {
+    chartWidth$.next(chartWidth);
+  }, [chartWidth]);
+
+  useEffect(() => {
+    severity$.next(severity);
+  }, [severity]);
+
+  return { chartsData, isLoading, error };
+}
+
+export function processFilters(filters: Filter[], query: Query) {
```

Comment:
Seems like a copy of the function from `swimlane_input_resolver`, probably you can just import it

Reply:
Updated here https://github.com/elastic/kibana/pull/94396/commits/5db8eb1637e3ee18c36b5db73aa06a10b1acd339